src/query/storages/system/src/tables_table.rs

```diff
@@ -245,6 +248,27 @@ where TablesTable<T, U>: HistoryAware
                                     db_name.push(database.clone());
                                 }
                             }
+                        } else if col_name == "table_id" {
+                            if let Scalar::String(tid) = scalar {
```

Comment:
use `check_number` to get the tid.

Reply:
How about as_u_int64()? NumberScalar support EnumAsInner

Reply:
Done